### PR TITLE
Add an option to make /hilight case sensitive

### PR DESCRIPTION
**Pull request by isundil**
See #421

### CODE REVIEW

**Comment by isundil** [comment]
I'm not sure I understand Travis, the outputed error seems to be non-blocking on the others pull-requests

**Comment by LemonBoy** [comment]
Looks fine code-wise with one minor remark about the choice of names, right now you use `-case` and `matchcase` in the command line and the config respectively and it'd be nice to have both named the same way.

**Comment by ailin-nemui** [comment]
Thanks for your contribution, if I'm not mistaken the code is lacking the regex case

**Comment by isundil** [comment]
Yes, I thought they were a regexp flag in preg, but obviously it doesn't

File: src/fe-common/core/hilight-text.c [hunk 1]
```diff
@@ -200,19 +204,26 @@ static int hilight_match_text(HILIGHT_REC *rec, const char *text,
...
 #endif
 	} else {
-		match = rec->fullword ?
-			stristr_full(text, rec->text) :
-			stristr(text, rec->text);
+		if (rec->case_sensitive) {
+			match = rec->fullword ?
+				strstr_full(text, rec->text) :
+				strstr(text, rec->text);
+		}
```
Comment (ailin-nemui) [inline src/fe-common/core/hilight-text.c]: please keep `} else {` style

**Comment by ailin-nemui** [comment]
thanks. please increase ABI version in src/common.h (change in struct/header file)

**Pull request merged**